packages/vsphere-cpi/charts/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: vsphere-cpi
+version: 1.0.0
+appVersion: 1.0.0
+description: vSphere CPI driver
+sources:
+  - https://github.com/kubernetes/cloud-provider-vsphere
+icon: file://logo.svg
+maintainers:
```

Comment:
Ideally we would add a Keyword with 'Infrastructure'

Reply:
added